scanner/cmd/scanner/main.go

```diff
@@ -4,14 +4,14 @@ import (
 	"context"
 	"flag"
 	"fmt"
-	"log"
+	golog "log"
 	"os"
 	"os/signal"
 	"path/filepath"
-	"syscall"
 
 	"github.com/quay/zlog"
 	"github.com/rs/zerolog"
+	"github.com/rs/zerolog/log"
```

Comment:
Nitpick: It would make more sense to rename the zero log package to `zlog` or something since someone reading the code would assume `log` means the default package `"log"`.